google/cloud/spanner_dbapi/cursor.py

```diff
@@ -258,9 +259,30 @@ def executemany(self, operation, seq_of_params):
 
         many_result_set = StreamedManyResultSets()
 
-        for params in seq_of_params:
-            self.execute(operation, params)
-            many_result_set.add_iter(self._itr)
+        if self.connection.autocommit or self.connection.use_mutations:
+            if classification == parse_utils.STMT_INSERT:
```

Comment:
I'm not sure whether this is still work in progress, but to be sure: This `if` statement has no `else`, which means that if a user executes an update or delete statement, nothing will be executed (in autocommit or use_mutations mode).